README.md

```diff
@@ -119,41 +122,160 @@ onSortingChange (sortOrder) {…}
 
 ### Action Bar
 
-#### Template
+#### Template Examples
 
 ```handlebars
-  {{frost-action-bar
-    controls=(array
-      (component 'frost-link'
-        priority='primary'
-        routes=detailLinkRoutes
-        size='medium'
-        text='Detail'
-      )
+{{! default moreActions behavior }}
+{{frost-action-bar
+  controls=(array
+    (component 'frost-link'
+      priority='primary'
+      routes=detailLinkRoutes
+      size='medium'
+      text='Detail'
     )
-    moreActions=(array
-      (component 'frost-button'
-        text='Extra button one'
-        hook='extraButtonOne'
-        onClick=(action 'doTheFirstExtraThing')
-      )
-      (component 'frost-button'
-        text='Extra button two'
-        hook='extraButtonTwo'
-        onClick=(action 'doTheSecondExtraThing')
-      )
+    (component 'frost-button'
+      text='Extra button one'
+      hook='extraButtonOne'
+      onClick=(action 'doTheFirstExtraThing')
     )
-    isLoading=isLoading
-    loadingText='Loading actions'
-    selectedItems=selectedItems
-  }}
+    (component 'frost-button'
+      text='Extra button two'
+      hook='extraButtonTwo'
+      onClick=(action 'doTheSecondExtraThing')
+    )
+    (component 'frost-button'
+      text='Extra button Three'
+      hook='extraButtonThree'
+      onClick=(action 'doTheSecondExtraThing')
```

Comment:
In these examples, extra buttons three through five all invoke `doTheSecondExtraThing`.  While legal, this might be confusing for a reader of the documentation.